src/app/modules/signatory-data/submodules/providersPage/utils/getTableData.ts

```diff
@@ -66,14 +72,22 @@ export const getTableData = (rawData, facetKey, codelist, typeSum) => {
           });
 
           providers.push([
-            facet.value || 'Not Provided',
             fpivot.value || 'Not Provided',
+            facet.value || 'Not Provided',
+            // tableType === 'provider'
```

Comment:
@stephanoshadjipetrou are you still going to use the commented out code? Or can it be removed